test/generic.jl

```diff
@@ -96,6 +97,18 @@ n = 5 # should be odd
     @testset "det with nonstandard Number type" begin
         elty <: Real && @test det(Dual.(triu(A), zero(A))) isa Dual
     end
+    if elty <: Int
+        @testset "det no overflow - triangular" begin
+            A = diagm([typemax(elty), typemax(elty)])
+            @test det(A) ≈ det(float(A))
```

Comment:
```suggestion
            @test det(A) == det(float(A))
```
Should be exactly equal, no?

Reply:
Yes